.eslintrc

```diff
@@ -1,21 +1,13 @@
-{
-    "env": {
-        "node": true
-    },
-    "extends": "eslint:recommended",
-    "rules": {
-        "no-mixed-requires": [2, { "allowCall": true }],
-        "quotes": [2, "single"],
-        "no-trailing-spaces": 2,
-        "indent": 2,
-        "linebreak-style": [2, "unix"],
-        "semi": [2, "always"],
-        "brace-style": [2, "1tbs", { "allowSingleLine": true }],
-        "keyword-spacing": 2,
-        "space-before-blocks": 2,
-        "space-before-function-paren": [2, { "anonymous": "always", "named": "never" }],
-        "no-mixed-spaces-and-tabs": 2,
-        "comma-spacing": [2, {"before": false, "after": true}],
-        "key-spacing": [2, {"beforeColon": false, "afterColon": true}]
-    }
-}
+extends:
+- "homeoffice/config/default"
+
+rules:
+  space-before-function-paren: 0
+  dot-notation: 0
+  func-names: 0
+  consistent-return: 0
+  no-else-return: 0
+  max-len: 0
```

Comment:
Once this is merged I'll raise a separate PR to go 4 => 2 space indentation and hopefully be able to get rid of the `max-len` disabling.